libbeat/scripts/Makefile

```diff
@@ -313,13 +320,72 @@ install-home:
 	install -d -m 755 ${HOME_PREFIX}/scripts/
 	install -m 755 ${ES_BEATS}/libbeat/scripts/migrate_beat_config_1_x_to_5_0.py ${HOME_PREFIX}/scripts/
 
-# Creates packer files in a beat
-.PHONY: create-packer
-create-packer:
-	python ${ES_BEATS}/libbeat/scripts/create_packer.py --es_beats=${ES_BEATS} --beat_path=${BEAT_DIR} --beat=${BEATNAME}
+
+# Prepares for packaging. Builds binaries and creates homedir data
+.PHONY: prepare-package
+prepare-package: build/upload
+	# cross compile on ubuntu
+	docker run --rm \
+		-v $(abspath ${ES_BEATS}/dev-tools/packer/xgo-scripts):/scripts \
```

Comment:
We're going to need to allow for the community beats to provide their own scripts folder / before_build script, otherwise they won't be able to install or adjust C deps.